Documentation/user-guides/getting-started.md

```diff
@@ -0,0 +1,148 @@
+# Prometheus Operator
+
+The mission of the Prometheus Operator is to make running Prometheus on top of Kubernetes as easy as possible, while preserving configurability as well as making the configuration Kubernetes native.
+
+To follow this getting started you will need a Kubernetes cluster you have access to. Let's give the Prometheus Operator a spin:
```

Comment:
At the beginning of this guide it's worth linking out to what an operator is again. Probably Brandon's initial blog post.